### ISSUE TITLE
IllegalArgumentException in YoutubeAudioSourceManager

### ISSUE THREAD

**Issue by Sgmaniac1255** [sedmelluq/lavaplayer#74]
when passing a straight videoID `example: dQw4w9WgXcQ` the manager calls my LoadFailed() method, 
however when passing in a URL `example: youtu.be/dQw4w9WgXcQ` the video loads just fine

while troubleshooting this, i get this stacktrace

```
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Something went wrong when looking up the track
	at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:40)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.dispatchItemLoadFailure(DefaultAudioPlayerManager.java:238)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$createItemLoader$0(DefaultAudioPlayerManager.java:227)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalArgumentException: No group with name <list>
	at java.util.regex.Matcher.getMatchedGroupIndex(Unknown Source)
	at java.util.regex.Matcher.group(Unknown Source)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadTrack(YoutubeAudioSourceManager.java:234)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadItemOnce(YoutubeAudioSourceManager.java:175)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadItem(YoutubeAudioSourceManager.java:119)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItemOnce(DefaultAudioPlayerManager.java:429)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItem(DefaultAudioPlayerManager.java:415)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$createItemLoader$0(DefaultAudioPlayerManager.java:219)
	... 4 more
```

i'm lead to believe that for some reason the SourceManager is trying to match the Regex of a playlist even though i'm passing in a videoID

**Comment by Sgmaniac1255** [comment]
come to find out this was broken and will be fixed by #70

**Comment by sedmelluq** [comment]
Fix merged.
